openlibrary/plugins/worksearch/subjects.py

```diff
@@ -169,6 +177,13 @@ def normalize_key(self, key):
     def process_key(self, key):
         return key
 
+def inject_availability(subject):
+    works = add_availability(subject.works)
+    for work in works:
+        ocaid = work.ia if work.ia else None
+        availability = work.get('availability', {}).get('status')
+    subject.works = works
+    return subject
```

Comment:
This probably is ok for functionality, but the naming continues the conflation we have between "subjects" and "Availability status" -- not sure if there is an easy way to improve this, either by renaming variables to be clearer, or adding comments to explain what is going on and how availability is not a real subject :)

Reply:
great point, will update var name to something clearer